docs/docs/contribute/contributing-guide.mdx

```diff
@@ -0,0 +1,53 @@
+# Contributing to the CLI for Microsoft 365
+
+We love to accept contributions. If you want to get involved in helping us grow the CLI for Microsoft 365, whether that is suggesting or adding a new command, extending an existing command, or updating our documentation, we would love to hear from you.
+
+Before you submit your first PR, please read the following guide. We'd hate to see you work on something that someone else is already working on, something that we agreed not to do, or something that doesn't match the project.
+
+Sharing is caring!
+
+## You have an idea for a New Command
+
+Awesome! Good ideas are invaluable for every product. Before you start hacking away, please check if there is no similar idea already listed in the [issue list](https://github.com/pnp/cli-microsoft365/issues). This ensures that your idea is unique and not being worked on. If you don't find a similar idea, create a new issue describing your idea. Once we agree on the feature scope and architecture, the feature will be ready for building. Don't hesitate to mention this in the issue if you'd like to build the feature yourself. If it's the first time you're building a command, see the [guidance article](https://github.com/pnp/cli-microsoft365/wiki/Adding-a-command) explaining in detail what you will need at a minimum.
```

Comment:
Right now, there's no specific page for this, but it would be great to have one on the website. Maybe we can create a new issue to address this. We'll review the existing wiki pages and make sure to remove any links if we decide to move the content there.